### PR TITLE
update test card design and add patient link

### LINKED ISSUE #3012: Redesign of Date/Time Fields on Test Card
### WHY?
Kenny has proposed the following changes to be more efficient with the space on the test card

**AS A** user of SR

**I WANT** the test card date/time windows to be smaller

**SO THAT** that the layout of the test card is cleaner

### Design

**BEFORE**

**AFTER**

**Existing date component from "Add person" workflow**

**Figma**
https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=5381%3A24437

**Spec**

https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=5381%3A24437

**Design tasks:**
- [ ] Move DOB to the top
- [ ] Move calendar icon inside the date field
- [ ] Tighten spacing within date and time components
- [ ] Condense headers to one title: 'Test date and time'
- [ ] Remove subtext headers in gray
- [ ] Change copy of checkbox text and header
- [ ] Tighten spacing between date and time components
- [ ] I recommend pairing with Kenny to review the frontend work together, once completed
- [ ] Expand the width of dropdown for devices and swab types with longer horizontal widths
- [ ] Ensure dropdown field content is sentence case.

### Translations
- [ ] Translations are required
- [ ] Translation request has been submitted
- [ ] Translations have been received

### PR DESCRIPTION
**Pull request by zdeveloper**
## Related Issue or Background Info

- fixed #3012 

## Changes Proposed
- updated test questionnaire modal onset symptom to use the new date-picker style (hoping this will also address #3074
image

### Before:
image

### After:
image
### UI
- [ ] Any changes to the UI/UX are approved by design 
- [ ] Any new or updated content (e.g. error messages) are approved by design 

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [x] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

## Cloud
- [x] DevOps team has been notified if PR requires ops support
- [x] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

File: frontend/src/app/testQueue/QueueItem.scss [hunk 1]
```diff
@@ -1,3 +1,130 @@
...
+.usa-checkbox__input {
+  height: 34px;
+  border: 2px solid #565c65;
+  margin-right: 12px;
+  max-width: 130px;
+  animation: 0.25s fadeIn;
+  animation-fill-mode: both;
+}
+
+@keyframes fadeIn {
```
Comment (nathancrtr) [inline frontend/src/app/testQueue/QueueItem.scss:46]: TIL about `@keyframes` - neat!

**Review by nathancrtr — APPROVED** [review]
The re-designed cards look awesome! Very impressive CSS-fu 😎

**Comment by kenieh** [comment]
Overall, looks great!

Three comments:
- I think we should change the Device + Swab type dropdown styling back to the default lighter ones (i.e. the one used in the "Add person" workflow). Right now, the dropdowns draw too much visual interest, when really these dropdowns should take up the least attention as the least important data points on the test card.

- Interested in seeing the static version of "Device" (see screenshot below for what I'm referencing) 
Screen Shot 2021-11-12 at 11 49 14 AM 
Is there a version of the test card available for viewing when there's only one device type for a facility? I realize I forgot to include explicit design requirements on covering this. I'm guessing we would want to cover this in a separate ticket?


- After clicking on the profile, the user's profile needs to have the option changed to go back to Conduct Test (see screenshot below). Right now it goes back to the "People" page instead of "Conduct test"Probably a new ticket we need to create @tim-skylight ?
Screen Shot 2021-12-07 at 3 35 15 PM

**Comment by zdeveloper** [comment]
> Overall, looking great!
> 
> Three comments:
> 
> 1. I think we should change the Device + Swab type dropdown styling back to the default lighter ones (i.e. the one used in the "Add person" workflow). Right now, the dropdowns draw too much visual interest, when really these dropdowns should take up the least attention as the least important data points on the test card.
> 2. Interested in seeing the static version of "Device" (see screenshot below for what I'm referencing)
> 
> Screen Shot 2021-11-12 at 11 49 14 AM
> 
> Is there a version of the test card available for viewing when there's only one device type for a facility? I realize I forgot to include explicit design requirements on covering this. If this wasn't covered for, I don't want to hold back this ticket any further and would recommend we draft another ticket to cover this.
> 3. After clicking on the profile, the user's profile needs to have the option changed to go back to Conduct Test (see screenshot below). Right now it goes back to the "People" page instead of "Conduct test"Probably a new ticket we need to create @tim-skylight ?
> 
> Screen Shot 2021-12-07 at 3 35 15 PM

Thanks for the review @kenieh , I am happy to make the styling back to the default lighter one.
I wanna push against the second ask, I think it would be very confusing to show swab types on the left and device types on the right. Also currently it auto selects the available options if only one is available, so it serves as a hint that they can change it if they add more devices. It will also add a lot more to this ticket, let's chat about it tomorrow
I agree with the thirds ask, we should definitely do it in a future ticket.,

**Comment by kenieh** [comment]
Oops my bad, I uploaded the wrong screenshots! Good catch, I didn't mean to switch the placements of Device and Swab type. These screenshots are more accurate:

Screen Shot 2021-12-08 at 8 30 55 AM

**Review by kenieh — APPROVED** [review]

**Review by nathancrtr — APPROVED** [review]
Re-approving after minor changes

**Pull request merged**